keyboards/nomu30/config.h

```diff
@@ -20,30 +20,17 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include "config_common.h"
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0xC0C0
-#define PRODUCT_ID      0x3000
+#define VENDOR_ID       0x524B // recompile keys
+#define PRODUCT_ID      0x4E31 // Nomu30
 #define DEVICE_VER      0x0001
-#define MANUFACTURER    Naoto Takai
-#define PRODUCT         nomu30
-#define DESCRIPTION     A 30% keyboard with ISO enter.
+#define MANUFACTURER    recompile keys
+#define PRODUCT         recompile keys Nomu30
```

Comment:
```suggestion
#define PRODUCT         Nomu30
```

Reply:
VIA requires that the vendor's name be included.

https://caniusevia.com/docs/configuring_qmk#change-product
> It is highly recommended that the value of PRODUCT is changed to be a combination of designer/vendor and the device name.

Reply:
I disagree with this, as it sometimes leads to duplication of the manufacturer string because some tools concatenate them together. Although there is no specific guidance in the USB spec, IMO the product string should *only* contain the product name, and the vendor string should *only* contain the vendor name.

Reply:
I just want to follow VIA's policies and don't have a strong opinion.

@Wilba6582 What are your thoughts on that?

Reply:
I took a quick look at VIA compatible keyboards and found that many of them don't have a vendor name. I will follow your review.